src/styles/blocks/price.scss

```diff
@@ -0,0 +1,17 @@
+.price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  &__label {
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main-text-color;
```

Comment:
The variable `$main-text-color` is used but not defined in this file. Ensure that it is defined or imported from another SCSS file to avoid compilation errors.